src/js/modules/infragistics.ui.editors.js

```diff
@@ -1079,9 +1079,17 @@
 					this._updateValue(newValue);
 					this._editorInput.val(this._getDisplayValue());
```

Comment:
change this line with something like:
 if (this._focused !== true) { 
   this._exitEditMode(); 
}